backend/package.json

```diff
@@ -5,17 +5,23 @@
   "license": "MIT",
   "description": "Backend server for the smart Insti App",
   "scripts": {
-    "start": "node ./bin/www"
+    "start": "nodemon ./bin/www"
```

Comment:
I think for the dev it is good to have nodemon , but for prod it is recomended to have node.